docs/tutorials/proposals/referral-program-proposal.md

```diff
@@ -0,0 +1,226 @@
+              }
+            ]
+          }
+      },
+      "closingTimestamp": 1111111111,
+      "enactmentTimestamp": 1111111155
+  }
+  }
+}'
+```
+
+</TabItem>
+<TabItem value="cmd-windows" label="Command line (Windows)">
+<TerminalInstructions />
+
+```
+vegawallet.exe transaction send --wallet YOUR_WALLETNAME --pubkey YOUR_PUBLIC_KEY --network NETWORK_NAME ^
+"{^
+\"proposalSubmission\": {^
+ \"rationale\": {^
+  \"title\": \"Referral proposal title\",^
+  \"description\": \"This is why I want to enact or replace referral program\"^
+ },^
+ \"terms\": {^
+ : {
+        \"updateReferralProgram\":^ {
+         \"changes\": {^
+            \"end_of_program_timestamp\": \"1234567890\",:^
+
+            \"window_length\": \"3\",
+            \"benefitTiers\": [
+              {
+                \"minimumRunningNotionalTakerVolume\": \"10000\",^
+                \"minimumEpochs\": \"6\",:^
+                \"referralRewardFactor\": \"0.001\",^
+                \"referralDiscountFactor\": \"0.001\"^
+              },
+              {
+                \"minimumRunningNotionalTakerVolume\": \"10000\",^
+                \"minimumEpochs\": \"6\",^
+                \"referralRewardFactor\": \"0.001\",^
+                \"referralDiscountFactor\": \"0.001\"^
+              }
+            ],
+            \"stakingTiers\": [
+              {
+                \"minimumStakedTokens\": \"100\",^
+                \"referralRewardMultiplier\": \"1.5\"^
+              },
+              {
+                \"minimumStakedTokens\": \"500\",^
+                \"referralRewardMultiplier\": \"2\"^
+              }
+            ]
+          }
+      },
+      \"closingTimestamp\": \"1111111111\",^
+      \"enactmentTimestamp\": \"1111111155\"
+  }^
+  }^
+}'^
+```
+</TabItem>
+</Tabs>
+
+## Voting
+All proposals are voted on by the community. To vote, community members need, at a minimum, the larger of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.referralProgram.minVoterBalance" formatter="governanceToken" suffix="tokens" hideName={true} /> or <NetworkParameter frontMatter={frontMatter} formatter="governanceToken" param="spam.protection.voting.min.tokens" suffix="tokens" hideName={true} /> associated to their Vega key.
+
+Your proposal will need [participation](../../concepts/governance.md#how-the-outcome-is-calculated) of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.referralProgram.requiredParticipation" formatter="percent" hideName={true} /> and a majority of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.referralProgram.requiredMajority" formatter="percent" hideName={true} />, so having community support is essential. 
+
+Proposers who invite feedback, engage with comments, and make revisions to meet the needs of the community are more likely to be successful.
+
+## Enactment
+If successful, the proposal will be enacted at the time you specify in the `enactmentTimestamp` field.
```

Comment:
Maybe clarify the benefits are not enabled un till the epoch after the enactment timestamp?